app/src/components/card/searchCard.tsx

```diff
@@ -38,60 +40,92 @@ const contentTypeTag = (result: SearchCardType) => {
       ? result.contentType === "Image" && result.containsMultipleCaptures
         ? "Multiple images"
         : result.contentType
-      : result.recordType;
+        ? result.contentType
+        : null
+      : result.recordType
+      ? result.recordType
+      : null;
 
   return (
-    <TagSet
-      tagSetData={[
-        { id: `type-${result.uuid}`, label: displayLabel ? displayLabel : "" },
-      ]}
-      type="filter"
-      sx={{ margin: 0 }}
-    />
+    displayLabel && (
+      <TagSet
+        tagSetData={[
+          {
+            id: `type-${result.uuid}`,
+            label: displayLabel ? displayLabel : "",
```

Comment:
You don't need this check for the fallback (it can also be `displayLabel || ""`) because if `displayLabel` is falsy, then `TagSet` won't be rendered at all.